src/misc/programs.cpp

```diff
@@ -891,76 +878,129 @@ void PROGRAMS_Destroy([[maybe_unused]] Section* sec)
 	call_program = CALLBACK_Allocate();
 	CALLBACK_Setup(call_program, &PROGRAMS_Handler, CB_RETF, "internal program");
 
-	// Cleanup -- allows unit tests to run indefinitely & cleanly
+	// TODO Cleanup -- allows unit tests to run indefinitely & cleanly
 	sec->AddDestroyFunction(&PROGRAMS_Destroy);
 
-	// listconf
-	MSG_Add("PROGRAM_CONFIG_NOCONFIGFILE", "No config file loaded!\n");
-	MSG_Add("PROGRAM_CONFIG_PRIMARY_CONF", "Primary config file: \n%s\n");
-	MSG_Add("PROGRAM_CONFIG_ADDITIONAL_CONF", "Additional config files:\n");
+	// List config
+	MSG_Add("PROGRAM_CONFIG_NOCONFIGFILE", "No config file loaded\n");
+	MSG_Add("PROGRAM_CONFIG_PRIMARY_CONF", "[color=white]Primary config file:[/reset]\n  %s\n");
+	MSG_Add("PROGRAM_CONFIG_ADDITIONAL_CONF", "[color=white]Additional config files:[/reset]\n  ");
+
 	MSG_Add("PROGRAM_CONFIG_CONFDIR",
-	        "DOSBox Staging %s configuration directory: \n%s\n\n");
+	        "[color=white]DOSBox Staging %s configuration directory:[/reset]\n  %s\n\n");
 
-	// writeconf
-	MSG_Add("PROGRAM_CONFIG_FILE_ERROR", "\nCan't open file %s\n");
-	MSG_Add("PROGRAM_CONFIG_FILE_WHICH", "Writing config file %s\n");
+	// Write config
+	MSG_Add("PROGRAM_CONFIG_FILE_ERROR", "\nCan't open file '%s'\n");
+	MSG_Add("PROGRAM_CONFIG_FILE_WHICH", "Writing config file '%s'\n");
 
-	// help
+	// Help
 	MSG_Add("SHELL_CMD_CONFIG_HELP_LONG",
-	        "Adjusts DOSBox Staging's configurable parameters.\n"
-	        "-writeconf or -wc without parameter: write to primary loaded config file.\n"
-	        "-writeconf or -wc with filename: write file to config directory.\n"
-	        "Use -writelang or -wl filename to write the current language strings.\n"
-	        "-r [parameters]\n"
-	        " Restart DOSBox, either using the previous parameters or any that are appended.\n"
-	        "-wcp [filename]\n"
-	        " Write config file to the program directory, dosbox.conf or the specified\n"
-	        " filename.\n"
-	        "-wcd\n"
-	        " Write to the default config file in the config directory.\n"
-	        "-l lists configuration parameters.\n"
-	        "-h, -help, -? sections / sectionname / propertyname\n"
-	        " Without parameters, displays this help screen. Add \"sections\" for a list of\n"
-	        " sections."
-	        " For info about a specific section or property add its name behind.\n"
-	        "-axclear clears the autoexec section.\n"
-	        "-axadd [line] adds a line to the autoexec section.\n"
-	        "-axtype prints the content of the autoexec section.\n"
-	        "-securemode switches to secure mode.\n"
-	        "-avistart starts AVI recording.\n"
-	        "-avistop stops AVI recording.\n"
-	        "-startmapper starts the keymapper.\n"
-	        "-get \"section property\" returns the value of the property.\n"
-	        "-set \"section property=value\" sets the value.\n");
+	        "Performs configuration management tasks and other miscellaneous actions.\n"
```

Comment:
Description line is too long for `help /all` command:

![description](https://github.com/dosbox-staging/dosbox-staging/assets/48332137/677ac6ad-909b-4501-ae68-6f9612a39c52)

Reply:
Ah, forgot about that!